src/stcal/skymatch/skystatistics.py

```diff
@@ -0,0 +1,136 @@
+"""
+The `skystatistics` module provides statistics computation class used by
+:py:func:`~stcal.skymatch.skymatch.match`
+and :py:class:`~stcal.skymatch.skyimage.SkyImage`.
+"""
+
+from copy import deepcopy
+
+# THIRD PARTY
+from stsci.imagestats import ImageStats  # type: ignore[import-untyped]
+
+__all__ = ["SkyStats"]
+
+
+class SkyStats:
+    """
+    This is a superclass build on top of
+    :py:class:`stsci.imagestats.ImageStats`. Compared to
```

Comment:
This comment is confusing because the class doesn't actually inherit from `ImageStats`

Reply:
Docs could be changed to explain that `SkyStats` delegates calls to an `ImageStats` object if that is more clear but the comment about persistence of settings is important as it is what differentiates it from `ImageStats` and I think it helps with performance.

Reply:
I agree that the comment itself explains an important distinction, and yes the simple wording change from "superclass built on top of" to "delegates calls to" would be clearer